app.js

```diff
@@ -0,0 +1,63 @@
+import isYes from '/src/is-yes.js';
+
+const actionButton = document.getElementById('action-button');
+const result = document.getElementById('quiz-result');
+
+function launchQuiz() {
+    let correctAnswers = 0;
+    let name = '';
+
+    alert('Hello! it is me Jaskier!');
+
+    const confirmed = confirm('Are you sure you want to take my test?');
+    if (confirmed === false) {
+        return;
+    }
+
+    const emptyString = '';    //why do i need this?
```

Comment:
You don't actually need this. Dani included this as an example of what an empty string looked like. Sorry for the confusion!